src/state/apis/nft/nftApi.ts

```diff
@@ -266,6 +263,7 @@ export const nftApi = createApi({
             if (!item.collection.assetId) return acc
             const cachedCollection = selectNftCollectionById(state, item.collection.assetId)
             if (cachedCollection?.isSpam) item.collection.isSpam = true
+            if (hasSpammyMedias(item.medias)) item.collection.isSpam = true
```

Comment:
we are already filtering spammy medias out above

Reply:
Nice catch, that's def. a redundant filtering fe4e5a1d06f9e7f0582813c14d59e535724666d1